src/server/pfs/server/trigger.go

```diff
@@ -1,10 +1,11 @@
 package server
 
 import (
+	"context"
+	"google.golang.org/protobuf/proto"
```

Comment:
This seems like a formatter issue? Usually the non stdlib imports are below.

Reply:
Yeah this is probably my IDE. I can fix that.